lib/web/mfa.go

```diff
@@ -211,3 +212,155 @@ func getDeviceUsage(reqUsage string) (proto.DeviceUsage, error) {
 
 	return deviceUsage, nil
 }
+
+type isMFARequiredDatabase struct {
+	// ServiceName is the database service name.
+	ServiceName string `json:"serviceName"`
+	// Protocol is the type of the database protocol.
+	Protocol string `json:"protocol"`
+	// Username is an optional database username.
+	Username string `json:"username,omitempty"`
+	// Name is an optional database name.
+	Name string `json:"name,omitempty"`
+}
+
+type isMFARequiredKube struct {
+	// Name is the name of the kube cluster.
+	Name string `json:"name"`
+}
+
+type isMFARequiredNode struct {
+	// Name can be node's hostname or UUID.
+	Name string `json:"name"`
+	// Login is the OS login name.
+	Login string `json:"login"`
+}
+
+type isMFARequiredWindowsDesktop struct {
+	// Name is the Windows Desktop server name.
+	Name string `json:"name"`
+	// Login is the Windows desktop user login.
+	Login string `json:"login"`
+}
+
+type isMfaRequiredRequest struct {
```

Comment:
```suggestion
type isMFARequiredRequest struct {
```

Initialisms, plus it matches the others.